### ISSUE TITLE
CDN Enabler does not replace URLs returned from wp_send_json

### ISSUE THREAD

**Issue by markusilmer** [keycdn/cdn-enabler#9]
Hi,
Woocommerce uses wp_send_json to return products on the cart. The Product image urls are not replaced with the CDN.
Do you have any suggestions for this problem?

**Comment by michaelrimbach** [comment]
(Ajax Mini Cart)

**Comment by svenba** [comment]
Feel free to open a pull request.

**Comment by chiedo** [comment]
Just an FYI, getting this plugin not working with WP-JSON responses could be a great move for KeyCDN. I added the plugin and set up the KeyCDN trial but then saw that it didn't work with WP-JSON so I've decided not to proceed.

I'll write a pull request if I get some time. But it's one of those things that I just wanted to pay a company a few dollars a month for without having to think about it.

But with WP-JSON not including the CDN URL, KeyCDN is pointless for my WordPress site since I'm only using it for the API... just an FYI!
